src/structures/GuildMember.js

```diff
@@ -80,6 +86,7 @@ class GuildMember extends Base {
       this.premiumSinceTimestamp = data.premium_since ? new Date(data.premium_since).getTime() : null;
     }
     if ('roles' in data) this._roles = data.roles;
+    this.communicationDisabledUntil = data.communication_disabled_until ? new Date(data.communication_disabled_until) : null;
```

Comment:
Shouldn't you use the `'prop' in obj` syntax here?

Reply:
I don't think so since the communication_disabled_until property passed by discord is either `null` or a ISO8601 timestamp
![image](https://user-images.githubusercontent.com/78901316/140608126-b59e601b-2e4d-479c-b588-e239a55be1c0.png)

Reply:
But it could be missing from partial GuildMember objects, like the ones sent in guild member update events

Reply:
I mean something like:
```js
if ('communication_disabled_until' in data) this.communicationDisabledUntil = new Date (..)
```

Reply:
i added it (see below)